packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -972,7 +974,7 @@ export class ApexDebug extends LoggingDebugSession {
     response: DebugProtocol.VariablesResponse,
     args: DebugProtocol.VariablesArguments
   ): Promise<void> {
-    response.success = true;
+    response.success = false;
```

Comment:
@DatGuyJonathan I've had it that way initially. However, in case of "errors" the variables request logs the error and returns an empty variables list. Thus, technically the response should be considered _successful_.